README.md

```diff
@@ -1,31 +1,74 @@
-# Reconstruction pipeline
-
-This pipeline is offering a solution to build full brain network models starting from standard structural MR scans.
-It is used to preprocess the MR scans in order to get actual files that are compatible with TVB. The result can be later uploaded in TVB or used independently for modeling. 
+<h1 align="center">
+  <br>
+  <a href="http://www.amitmerchant.com/electron-markdownify"><img src="https://cdn.evbuc.com/eventlogos/50084740/tvbtaglinecrop.jpg" alt="The Virtual Brain" width="300"></a>
+  <br>
+  Reconstruction pipeline
+  <br>
+</h1>
+<h4 align="center">This pipeline is offering a solution to build full brain network models starting from standard structural MR scans.</h4>
+<p align="center">
+  <a href="https://pegasus.isi.edu/downloads/">
+    <img src="https://img.shields.io/badge/Pegasus%20WMS-4.9.3-%2349A13D"
+         alt="Gitter">
+  </a>
+  <a href="https://neurostars.org/">
+	  <img src="https://img.shields.io/badge/Chat-on%20Neurostars-%23673B90">
```

Comment:
I would put the link towards tvb official forum: https://groups.google.com/forum/#!forum/tvb-users instead of neurostars

Reply:
The rest looks good from my side. Thank you for your contribution!

Reply:
Okay sure, sir.